Update Terms of Service for Strong Verification (#4448)
Couchers-org/couchers: app/backend/resources/terms_of_service.md

You must agree to these Terms to access the Couchers.org services and create an account.

*This is version 2.3 of the Terms of Service, last updated on 11th of June, 2024.*

# 1. Service provider

Access to the Couchers.org platform ("platform", "service") is provided to you by **Couchers, Inc.** ("Couchers.org", "we", "us"), a 501(c)(3) non-profit organization incorporated in the United States.

# 2. Eligibility

You must be at least 18 years old and a natural person to access the service.

> You are not permitted to have more than one account at a time on the service. If you are restricted from accessing the platform via a ban, suspension, or any other method, you may not create a new account. If you are in good standing, you may delete your account and sign up again later.

Reviewer: ```suggestion
Users are permitted to have only one account on Couchers.org. Creating, maintaining, accessing, or using multiple accounts is strictly prohibited under any circumstances. The only exception to this rule is if a user voluntarily deletes their account, they may return and create a new account. Users whose accounts have been suspended, banned, or restricted are not allowed to create new accounts. Violation of this policy will result in the termination of all associated accounts.

Accounts on Couchers.org are intended to foster genuine, platonic connections and community engagement within the context of travel and hospitality. Using accounts for purposes, including but not limited to data mining, surveillance, dating, sexual or romantic encounters, promotional endeavors, spamming, or any other activities that contradict the spirit of the platform or the community guidelines is strictly prohibited. Violations will lead to immediate account suspension or termination.
```

Author: I think the original is better: firstly using "You", since "user" is kind of the same as an account. So then we'd have to explain that user = you.

Also I think the "strictly prohibited..." wording is a bit too negative and draconian. I would prefer a more positive and neutral tone.

Author: sorted offline

Couchers.org provides a platform for users to contact one another and organize activities together. You agree to not hold us liable or responsible for user-created content or for interactions between users, including online and in person. You are solely liable and responsible for your actions on the platform, including any interactions with other users and any content that you create.

> Although we are not legally responsible for user-created content, we actively moderate the platform to remove any content that is harmful, deceitful, or otherwise does not adhere to our rules. If you believe some content does not adhere to our rules, please report it to us via support@couchers.org, or via the flagging feature on the platform.

Reviewer: ```suggestion
If you believe some content does not adhere to our rules, please report it to us via safety@couchers.org, or via the flagging feature on the platform.
```

> Although we are not legally responsible for user-created content, we actively moderate the platform to remove any content that is harmful, deceitful, or otherwise does not adhere to our rules. If you believe some content does not adhere to our rules, please report it to us via support@couchers.org, or via the flagging feature on the platform.

Reviewer: ```suggestion
Although we are not legally responsible for user-created content, we actively moderate the platform to remove any content that is harmful, deceitful, or otherwise does not adhere to our rules. If you believe some content does not adhere to our rules, please report it to us via safety@couchers.org, or via the flagging feature on the platform.
```

# 6. Our service providers

Our platform is hosted on Amazon Web Services (AWS), and we use a variety of their products to provide this service to you, including hosting our servers, storing our backups, and sending emails.

We use Google Analytics for analytics purposes. We use Google Workspace for our email hosting, Sentry for automatic bug reports, and GitHub for code hosting and bug reports.

We may add additional service providers as needed, and will make an effort to update this document.

# 7. Termination

> If you are in breach of these terms or any other rules, guidelines, or policies in place, we may, at our discretion, suspend or terminate your account.

Reviewer: ```suggestion
If you are in breach of these terms or any other rules, guidelines, or policies in place, we may, at our discretion, suspend or terminate your account. By using our services, you agree that we are not liable for any consequences or damages arising from a suspension or termination of your account.
```

Author: I'm not sure what liability there would be?

Author: resolved offline

We use a trusted third-party application called [Iris ID](https://passportreader.app/) by Iris Development AB to read your passport if you choose to complete Strong Verification. The app processes your data and verifies it before passing it back to our server. As per their [Privacy policy](https://passportreader.app/privacy), they will delete your personal information within 24 hours. By completing Strong Verification, you agree to be bound by their [Privacy policy](https://passportreader.app/privacy) and [Terms & conditions](https://passportreader.app/terms).

When you complete Strong Verification, the data retrieved from your passport is encrypted with an asymmetric keypair in such a way that the server can no longer access it. There is a mechanism for this data to be decrypted and retrieved manually, but the key is stored separately offline and requires human intervention to use. The following information is additionally stored on the server for continued verification purposes: country of nationality, sex, date of birth, passport expiry date, and the last three digits of the passport number.

> You can request for this Strong Verification data (except for the minimal anti-abuse data) to be deleted from the server at any time.

Reviewer: whose servers?

Author: ours

Author: they're not stored on Iris's servers

When you complete Strong Verification, the data retrieved from your passport is encrypted with an asymmetric keypair in such a way that the server can no longer access it. There is a mechanism for this data to be decrypted and retrieved manually, but the key is stored separately offline and requires human intervention to use. The following information is additionally stored on the server for continued verification purposes: country of nationality, sex, date of birth, passport expiry date, and the last three digits of the passport number.

You can request for this Strong Verification data (except for the minimal anti-abuse data) to be deleted from the server at any time.

> The minimal anti-abuse data of nationality, passport expiry date, and the last three digits of the passport number is stored for all successful verifications and cannot be removed. This data is used for safety and moderation purposes in order to avoid banned users from recreating accounts. We believe this is an appropriate compromise between providing a safe platform for our users, while not storing sensitive personal information beyond what you wish to share with the platform and other users.

Reviewer: ```suggestion
The minimal anti-abuse data of nationality, passport expiry date, and the last three digits of the passport number is stored for all successful verifications and cannot be removed. This data is used for safety and moderation purposes, for example, to prevent banned users from creating new accounts. We believe this is an appropriate compromise between providing a safe platform for our users, while not storing sensitive personal information beyond what you wish to share with the platform and other users.
```
